### PR TITLE
Update input parameters dataset workflows

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Describe your changes
The input parameters in the workflow after the change of PCA - hvg order were not adapted.
This PR has been broadend to update the daaset loaders to include at least a `.var.feature_name` to each dataset and also a `.var.feature_id` if available in the original dataset.
Also fixed a docker container build fail (quarto install) `create_task_readme`.

## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [x] Minor changes
  - [x] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [x] CI Tests succeed and look good!

### CODE REVIEW

File: src/datasets/loaders/openproblems_v1_multimodal/script.py [hunk 1]
```diff
@@ -120,6 +120,29 @@
...
+        mod1.var["feature_name"] = mod1.var.index
+        mod1.var["feature_name"] = mod1.var.index
+    elif par["var_feature_name"] in mod1.var:
+        mod2.var["feature_name"] = mod2.var[par["feature_name"]]
+        mod2.var["feature_name"] = mod2.var[par["feature_name"]]
+    else:
+        print(f"Warning: key '{par['var_feature_name']}' could not be found in adata.var.", flush=True)
+
+print("Setting .var['feature_id']", flush=True)
+if par["var_feature_id"]:
```
Comment (rcannood) [inline src/datasets/loaders/openproblems_v1_multimodal/script.py:135]: same here

File: src/datasets/loaders/openproblems_v1_multimodal/script.py [hunk 2]
```diff
@@ -120,6 +120,29 @@
...
+print("Setting .var['feature_name']", flush=True)
+if par["var_feature_name"]:
+    if par["var_feature_name"] == "index":
+        mod1.var["feature_name"] = mod1.var.index
+        mod1.var["feature_name"] = mod1.var.index
+    elif par["var_feature_name"] in mod1.var:
+        mod2.var["feature_name"] = mod2.var[par["feature_name"]]
+        mod2.var["feature_name"] = mod2.var[par["feature_name"]]
+    else:
+        print(f"Warning: key '{par['var_feature_name']}' could not be found in adata.var.", flush=True)
```
Comment (rcannood) [inline src/datasets/loaders/openproblems_v1_multimodal/script.py:132]: Should this not be:

```bash
    if par["var_feature_name"] == "index":
        mod1.var["feature_name"] = mod1.var.index
        mod2.var["feature_name"] = mod2.var.index
    else:
        if par["var_feature_name"] in mod1.var:
            mod1.var["feature_name"] = mod1.var[par["feature_name"]]
        else:
            print(f"Warning: key '{par['var_feature_name']}' could not be found in adata_mod1.var.", flush=True)
        if par["var_feature_name"] in mod2.var:
            mod2.var["feature_name"] = mod2.var[par["feature_name"]]
        else:
            print(f"Warning: key '{par['var_feature_name']}' could not be found in adata_mod2.var.", flush=True)
```

?

Comment (KaiWaldrant) [reply src/datasets/loaders/openproblems_v1_multimodal/script.py]: you are correct !

**Review by rcannood — APPROVED** [review]
LGTM! The `cell_cycle_conservation` component doesn't seem to be working however -- @mumichae could you take a look?

**Pull request merged**